src/modules/Lan.rb

```diff
@@ -968,15 +972,16 @@ def dhcp_ntp_servers
     #
     # @param id [Symbol] Network configuration ID
     # @return [Y2Network::Config,nil] Network configuration with the given ID or nil if not found
-    def find_config(id: :system)
-      configs.find { |c| c.id == id }
+    def find_config(id)
+      configs[id]
```

Comment:
Is possible to have here something like lazy loading here? I think it will be easier for routing module to simply ask for config and if it is not yet read, read it. And same with yast_config. If not available, copy system one.

Reply:
For the time being, I have decided to read the current configuration with the rest of network related values in `#Import` or `#Read`.